hac-client/src/pages/collection_viewer/collection_viewer.rs

```diff
@@ -60,16 +61,18 @@ impl PaneFocus {
             PaneFocus::Sidebar => PaneFocus::ReqUri,
             PaneFocus::ReqUri => PaneFocus::Editor,
             PaneFocus::Editor => PaneFocus::Preview,
-            PaneFocus::Preview => PaneFocus::Sidebar,
+            PaneFocus::Preview => PaneFocus::SampleResponse,
+            PaneFocus::SampleResponse => PaneFocus::Sidebar,
```

Comment:
personally, I think Sample responses may be best suited to live within Pretty/Raw response viewers, i'm not sure how we should allow the user to switch between actual current response and saved sample responses, but I think a whole new pane for this is not really needed, what you think?